policyengine_us/parameters/gov/states/sc/tax/income/credits/cdcc/amount.yaml

```diff
@@ -0,0 +1,14 @@
+description: South Carolina provides this amount under the child and dependent care credit.
+values:
+  2021-01-01: 210
+metadata:
+  reference:
+    - title: SC-1040 2021
+      href: https://dor.sc.gov/forms-site/Forms/SC1040_2021.pdf
+    - title: Instructions for Form SC-1040 (line 11)
+      href: https://dor.sc.gov/forms-site/Forms/IITPacket_2021.pdf#page=22
+    - title: South Carolina Legal Code | SECTION 12-6-3380
```

Comment:
$210 is not in the legal code. Please make sure to only cite relevant references.